kuksa_apps/kanto/config/edge_client.ini

```diff
@@ -0,0 +1,11 @@
+[SERVER]
+# host: default 127.0.0.1
+host = 192.168.56.51
```

Comment:
It's not a good idea to hardcode random local network IPs into the default configuration. Please, keep the `127.0.0.1` as the default one.

Reply:
Due to git-related mistakes I made, some early-stage development changes were committed. With the last commit (#325) I pushed the changes related to your comments @konstantina-gramatova :)

This section was replaced by 'kuksa_val' and a field called 'ip' holding localhost as value.